woocommerce/woocommerce-ios
# [Analytics Hub] Update networking to load data in parallel

Closes: https://github.com/woocommerce/woocommerce-ios/issues/8318.

## Description

This PR updates data loading flow to prevent waiting on all requests before updating any cards.

### How

- Update `TopPerformersView` to have its own viewmodel and loading state
- Update `AnalyticsProductCard` to accept 2 separate viewmodels
- Update data flow to trigger networking requests independently

## Testing

1. Build and run the app in debug/alpha mode.
2. On the dashboard view tap the "See more" button.
3. Scroll to the bottom of a list.
4. Confirm that top performers list has its own loading state and updates separately from the rest of stats.

## Video

> Revenue and orders cards are hidden to focus on products card

https://user-images.githubusercontent.com/3132438/206773285-0fde02e4-a885-4e85-9aff-6f5b8961042e.mp4

---
